From 58179fadeacd3134153d44acbc458ef7e94b274d Mon Sep 17 00:00:00 2001
From: Loic Poulain <loic.poulain@intel.com>
Date: Tue, 21 Oct 2014 02:43:52 +0200
Subject: [PATCH] ACPI / platform: add ID for Broadcom BCM43340 Bluetooth chip

This ID is used on Baytrail boards such as ACER Vespa8.

Signed-off-by: Loic Poulain <loic.poulain@intel.com>
---
 drivers/acpi/acpi_platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/acpi/acpi_platform.c b/drivers/acpi/acpi_platform.c
index 9ca745a..ea31e74 100644
--- a/drivers/acpi/acpi_platform.c
+++ b/drivers/acpi/acpi_platform.c
@@ -36,6 +36,7 @@ static const struct acpi_device_id acpi_platform_device_ids[] = {
 	{ "BCM2E1A" },
 	{ "BCM2E39" },
 	{ "BCM2E3D" },
+	{ "BCM2E3A" },
 	{ "RTL8723" },
 	{ "OBDA8723" },
 	{ "BCM4752E" },
-- 
1.9.1

